deploy/src/erc_to_native/home.js

```diff
@@ -62,13 +70,31 @@ async function deployHome() {
   homeNonce++
 
   console.log('\ninitializing Home Bridge Validators with following parameters:\n')
-  console.log(
-    `REQUIRED_NUMBER_OF_VALIDATORS: ${REQUIRED_NUMBER_OF_VALIDATORS}, VALIDATORS: ${VALIDATORS}`
-  )
   bridgeValidatorsHome.options.address = storageValidatorsHome.options.address
-  const initializeData = await bridgeValidatorsHome.methods
-    .initialize(REQUIRED_NUMBER_OF_VALIDATORS, VALIDATORS, HOME_VALIDATORS_OWNER)
-    .encodeABI({ from: DEPLOYMENT_ACCOUNT_ADDRESS })
+
+  let initializeData
+
+  if (useRewardableValidators) {
+    console.log(
+      `REQUIRED_NUMBER_OF_VALIDATORS: ${REQUIRED_NUMBER_OF_VALIDATORS}, VALIDATORS: ${VALIDATORS}, VALIDATORS_REWARD_ACCOUNTS: ${VALIDATORS_REWARD_ACCOUNTS}, HOME_VALIDATORS_OWNER: ${HOME_VALIDATORS_OWNER}`
```

Comment:
can we log `VALIDATORS` and `VALIDATORS_REWARD_ACCOUNTS` in the following manner?
```
VALIDATORS
==========
1: 0x..., reward address 0x...
2: 0x..., reward address 0x...
3: 0x..., reward address 0x...
```

Reply:
Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/134/commits/e0bf48958fef5f0706f6cf79b0818d987f8c1030